virtinst/install/installer.py

```diff
@@ -396,6 +396,14 @@ def _prepare(self, guest, meter):
         elif self._cloudinit_data:
             self._prepare_cloudinit(guest, meter)
 
+        # The information needed to perform an unattended installation might
+        # be passed to the guest in the form of a CDROM, which can potentially
+        # be attached via SCSI. libvirt will automatically add the SCSI
+        # controller for us, but depending on the architecture its choice of
+        # model might not be usable by the guest OS. Try to add a virtio-scsi
+        # controller for the best chance of the CDROM actually being picked up
+        guest.add_virtioscsi_controller()
+
```

Comment:
This doesn't look like the correct place. The cdrom device is added in function `_add_unattended_install_cdrom_device()` so we can add the controller there as well.

Reply:
You're absolutely right. Changed it.